database.py

```diff
@@ -0,0 +1,6 @@
+from pymongo import MongoClient
+from pymongo.collection import Collection
+
+client=MongoClient()
+
+coffees_collection : Collection = client.db.coffees
```

Comment:
please remember to format your files with black, you can set Visual Studio code to do it for you on every file save https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0